### PR TITLE
Optimize new conversation creation

### LINKED ISSUE #1479: Optimize new to start conversation latency
Likely want to stay on the same page and replace the URL and switch to the same components as the conversation page

### PR DESCRIPTION
**Pull request by spolu**
Fix #1479 
Fix #1489 

This is not perfect but I believe it optimizes a fair amount.

When a conversation is started we immediately clear the UI once the convo is created before sending the first message. We then start the router.push as soon as possible.

The rest is cache / swr and streaming kicking in.

We also added the support to posting a first user message at converstaion creation time

### CODE REVIEW

File: front/pages/w/[wId]/assistant/new.tsx [hunk 1]
```diff
@@ -110,50 +130,68 @@ export default function AssistantNew({
...
+      isWideMode={conversation ? true : false}
       gaTrackingId={gaTrackingId}
       topNavigationCurrent="lab"
       subNavigation={subNavigationLab({ owner, current: "assistant" })}
+      titleChildren={
+        conversation && (
+          <ConversationTitle
+            title={conversation.title || ""}
+            shareLink={`${baseUrl}/w/${owner.sId}/assistant/${conversation.sId}`}
+            // onDelete={() => {}}
```
Comment (fontanierh) [inline front/pages/w/[wId]/assistant/new.tsx:148]: nit

**Review by fontanierh — DISMISSED** [review]
lgtm

File: front/pages/w/[wId]/assistant/new.tsx (hunk 1 above)
Comment (spolu) [reply front/pages/w/[wId]/assistant/new.tsx:148]: we don't quite have a great story for that yet

**Review by fontanierh — APPROVED** [review]

**Pull request merged**